apps/backoffice-v2/src/pages/Entity/hooks/useTasks/selectors/selectDirectorsDocuments.ts

```diff
@@ -0,0 +1,8 @@
+import { AnyObject } from '@ballerine/ui';
+
+export const selectDirectorsDocuments = (workflow: unknown): AnyObject[] =>
+  //@ts-ignore
+  ((workflow?.context?.entity?.data?.additionalInfo?.directors as AnyObject[]) || [])
+    .map(director => director.additionalInfo?.documents)
+    .filter(Boolean)
+    .flat() || ([] as AnyObject[]);
```

Comment:
Seems like a place likely to break.
```suggestion
    ?.map(director => director.additionalInfo?.documents)
    ?.filter(Boolean)
    ?.flat() || ([] as AnyObject[]);
```